src/kube_vxlan_controller_pod_reg.erl

```diff
@@ -117,10 +117,11 @@ terminate(_Reason, _State) ->
 %%% Internal functions
 %%%===================================================================
 
-start_pod(Id, Event, Config, _State) ->
+start_pod(Id, Event, Config, State) ->
     case ?PodSup:start_pod(Id, Event, Config) of
 	{ok, Pid} ->
-	    ets:insert_new(?SERVER, {Id, Pid, undefined});
+            ets:insert_new(?SERVER, {Id, Pid, undefined}),
```

Comment:
this need to be be removed, the register action will handle that.

Reply:
yes, removed